iothub/device/tests/ClientSettings/TransportSettingsTests.cs

```diff
@@ -88,49 +111,56 @@ public void MqttTransportSettings_RespectsCtorParameter(IotHubClientTransportPro
         }
 
         [TestMethod]
-        [ExpectedException(typeof(ArgumentOutOfRangeException))]
-        public void AmqpConnectionPoolSettings_UnderMinPoolSize()
+        [ExpectedException(typeof(ArgumentException))]
+        public async Task IotHubDeviceClient_NullX509Certificate()
         {
-            _ = new AmqpConnectionPoolSettings { MaxPoolSize = 0 };
+            // arrange
+            const string hostName = "acme.azure-devices.net";
+            var authMethod = new ClientAuthenticationWithX509Certificate(null, "device1");
+            var options = new IotHubClientOptions(new IotHubClientAmqpSettings { PrefetchCount = 100 });
+
+            // act
+            await using var deviceClient = new IotHubDeviceClient(hostName, authMethod, options);
         }
 
         [TestMethod]
-        public void AmqpConnectionPoolSettings_MaxPoolSizeTest()
+        public async Task IotHubDeviceClient_FileUploadTransport_Proxy()
         {
             // arrange
-            const uint maxPoolSize = AmqpConnectionPoolSettings.AbsoluteMaxPoolSize;
+            const string hostName = "acme.azure-devices.net";
+            var authMethod = new ClientAuthenticationWithX509Certificate(new X509Certificate2(), "device1");
+            var options = new IotHubClientOptions(new IotHubClientAmqpSettings { PrefetchCount = 100 });
+            options.FileUploadTransportSettings = new IotHubClientHttpSettings()
+            {
+                Proxy = new WebProxy(),
+            };
 
             // act
-            var connectionPoolSettings = new AmqpConnectionPoolSettings { MaxPoolSize = maxPoolSize };
-
-            // assert
-            Assert.AreEqual(maxPoolSize, connectionPoolSettings.MaxPoolSize, "Should match initialized value");
+            await using var deviceClient = new IotHubDeviceClient(hostName, authMethod, options);
```

Comment:
What's the validation here? That it doesn't throw? If so, could use FluentAssertions to make that clear.